contracts/governance/extensions/GovernorVotesQuorumFractional.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "./GovernorVotes.sol";
+
+/**
+ * @dev Extension of {Governor} for voting weight extraction from an {ERC20Votes} token and a quorum expressed as a
+ * fraction of the total supply.
+ *
+ * _Available since v4.3._
+ */
+abstract contract GovernorVotesQuorumFractional is GovernorVotes {
```

Comment:
Suggested renaming:

- `QuorumFractional` -> `QuorumFraction`
- "quorum ratio" -> "quorum numerator"
- "quorum ratio max" -> "quorum denominator"

I'd get rid of the term "ratio" because I don't think it has an obvious interpretation. (The ratio of what to what?)

Did you consider a single getter returning a tuple (numerator, denominator)? It sounds like a reasonable way to represent a rational number.